### PR TITLE
Created rails react

### PR DESCRIPTION
**Pull request by TuyishimireEric**
# Set up a "Hello world" project as one app with Webpack
in this project, I implemented a connection between a Ruby on Rails back-end and React front-end.
### Highlight :recycle: 

-  Created a new Rails app called 'hello-rails-react'.
-   Used the commend rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
-   Verify that the jsbundling-rails gem is added to your Gemfile.
-   Initialize your project with Git.
-   Install needed node dependencies as described in the previous lesson's tutorial.
-   Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
-   Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
-   Make sure that your project has a Postgres database set up.
-   Create a table for storing your messages and create 5 different greetings.
-   Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
-   Create a static view that will be the root of your app.
-   Create the App component with react-router and render it in your static view.
-   Create the Greeting component that will display a greeting. Set it up as a route in your App component.
-   Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
-   Display the random greeting in your Greeting component.

### CODE REVIEW

File: app/views/root/index.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<div id="root">
```
Comment (HAMMAS-SALEEM) [inline app/views/root/index.html.erb:1]: image

- [ ] Your API is working and giving random quote :blush: but your front end part is not showing anything. Kindly fix it :blush: Your react app is not working properly :thinking:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse curriculum.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [ ] You have a link to your license file but this link is not working 😨 Kindly fix it to make your readme file descriptive 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @TuyishimireEric :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/root/index.html.erb (hunk 1 above)
Comment (TuyishimireEric) [reply app/views/root/index.html.erb:1]: Thank you @HAMMAS-SALEEM for reviewing my project, :+1: 
but on my side I am running ```rails db:seed``` and ```./bin/dev``` in my terminal and it's working on my side
image

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @TuyishimireEric 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Follow Gitflow
✔️ Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TuyishimireEric** [comment]
Thank you for approving my project :pray:

**Pull request merged**